lib/anonymizer.js

```diff
@@ -0,0 +1,320 @@
+const multiparty = require('multiparty');
+const cheerio = require('cheerio');
+const fs = require('fs');
+const path = require('path');
+
+// load anonymizerConfig
```

Comment:
Remove commented out code if not used

Reply:
This code is still there @AJun01, please remove it.